src/utils/useTimelapseCounter.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useEffect, useState } from 'react';
+
+const useTimelapseCounter = initialTimestamp => {
```

Comment:
suggestion(minor): Move this file to the `hooks` folder.

Reply:
Thanks for the suggestion, moved to hooks folder. Waiting review.🙂